M2/Macaulay2/packages/DGAlgebras.m2

```diff
@@ -1,26 +1,64 @@
--- -*- coding: utf-8 -*-
+-*- coding: utf-8 -*-
 newPackage("DGAlgebras",
      Headline => "Data type for DG algebras",
-     Version => "1.1.0",
+     Version => "1.1.1--with fix of killCycles and new code and docs for displayBlockDiff and blockDiff",
      Date => "August 20, 2020",
      Authors => {
 	  {Name => "Frank Moore",
 	   HomePage => "http://www.math.wfu.edu/Faculty/Moore.html",
-	   Email => "moorewf@wfu.edu"}},
+	   Email => "moorewf@wfu.edu"},
+       {Name => "Some documentation added by Daniel Rostamloo and David Eisenbud"}},
```

Comment:
We'd rather you use the [Contributors](https://faculty.math.illinois.edu/Macaulay2/doc/Macaulay2/share/doc/Macaulay2/Macaulay2Doc/html/___Contributors.html) key in the documentation. This "name" is going to mess up the list of package [authors](https://faculty.math.illinois.edu/Macaulay2/doc/Macaulay2/share/doc/Macaulay2/Macaulay2Doc/html/_authors_spof_sp__Macaulay2_sppackages.html).